SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebuggerEditor.cpp

```diff
@@ -107,52 +69,44 @@ bool ASpatialDebuggerEditor::AllowWorkerBoundaries() const
 	}
 		
 	const ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(World->GetWorldSettings());
-
 	const bool bIsMultiWorkerEnabled = WorldSettings != nullptr && WorldSettings->IsMultiWorkerEnabled();
-
-	return bIsMultiWorkerEnabled && !bIsGameInProgress;
-	
+	const bool bIsSpatialNetworkingEnabled = GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking();
+	return bIsMultiWorkerEnabled && bIsSpatialNetworkingEnabled;
+#else
+	return false;
+#endif
 }
 
 void ASpatialDebuggerEditor::InitialiseWorkerRegions()
 {
-	ULayeredLBStrategy* LoadBalanceStrategy = NewObject<ULayeredLBStrategy>();
-	LoadBalanceStrategy->Init(GetWorld());
-	LoadBalanceStrategy->SetVirtualWorkerIds(1, LoadBalanceStrategy->GetMinimumRequiredWorkers());
+#if WITH_EDITOR
+	WorkerRegions.Empty();
+
+	ULayeredLBStrategy* LoadBalanceStrategy = NewObject<ULayeredLBStrategy>(this);
+	LoadBalanceStrategy->Init();
+	const ULayeredLBStrategy* LayeredLBStrategy = Cast<ULayeredLBStrategy>(LoadBalanceStrategy);
```

Comment:
This is already a ULayeredLBStrategym you just created it above, this variable can go

Reply:
Sure. Removed unnecessary variable.